src/mixed_script.rs

```diff
@@ -67,14 +65,50 @@ impl From<&'_ str> for AugmentedScriptSet {
 impl Default for AugmentedScriptSet {
     fn default() -> Self {
         AugmentedScriptSet {
-            base: ScriptExtension::Single(Script::Common),
+            base: ScriptExtension::from(Script::Common),
             hanb: true,
             jpan: true,
             kore: true,
         }
     }
 }
 
+impl Debug for AugmentedScriptSet {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        if self.is_empty() {
+            write!(f, "AugmentedScriptSet{{∅}}")?;
+        } else if self.is_all() {
+            write!(f, "AugmentedScriptSet{{ALL}}")?;
+        } else {
+            write!(f, "AugmentedScriptSet{{")?;
+            let mut first_entry = true;
+            let hanb = if self.hanb { Some("Hanb") } else { None };
+            let jpan = if self.jpan { Some("Jpan") } else { None };
+            let kore = if self.kore { Some("Kore") } else { None };
+            for writing_system in None.into_iter().chain(hanb).chain(jpan).chain(kore) {
```

Comment:
we can just chain an `base.iter.map(Script::short_name)` here

Reply:
Indeed! Thanks